apps/minifront/src/state/ibc-out.ts

```diff
@@ -199,15 +199,16 @@ export const ibcValidationErrors = (state: AllSlices) => {
   return {
     recipientErr: !state.ibcOut.destinationChainAddress
       ? false
-      : !validateUnknownAddress(state.ibcOut.chain, state.ibcOut.destinationChainAddress),
+      : validateUnknownAddress(state.ibcOut.chain, state.ibcOut.destinationChainAddress),
     amountErr: !state.ibcOut.selection
       ? false
       : amountMoreThanBalance(state.ibcOut.selection, state.ibcOut.amount),
   };
 };
 
 /**
- * we don't know what format foreign addresses are in. so this only checks:
+ * Matches the given address to the chain's address prefix. Returns `true` if the address is not valid.
```

Comment:
Hmm, think there could be a bug in `validateUnknownAddress`. It should return `true` if the address _is_ valid. That is consistent with the first line:

```
if (!chain || address === '') return false; // not valid = false
```

We should rename this function to: `unknownAddrIsValid()` to add clarity.

Reply:
Right, updated!